docs/provisioner/01-01-provisioner-overview.md

```diff
@@ -3,19 +3,15 @@ title: Overview
 type: Overview
 ---
 
-The Runtime Provisioner is a Compass component responsible for provisioning, installing, and deprovisioning clusters with Kyma (Kyma Runtimes). The relationship between clusters and Runtimes is 1:1.
+The Runtime Provisioner is a Compass component responsible for provisioning, installing, and deprovisioning clusters with Kyma (Kyma Runtimes). The relationship between clusters and Runtimes is 1:1. The Runtime Provisioner is registered in the Director as an integration system. 
```

Comment:
`Runtime Provisioner` vs. `Provisioner`? We need more consistency throughout the Compass docs. Also: `Director` vs. `the Director`?